static/directives/repo-view/repo-panel-mirroring.html

```diff
@@ -361,124 +361,125 @@ <h4>Status</h4>
 
     <!-- Modal: Change mirroring sync date dialog -->
     <div class="cor-confirm-dialog"
-         dialog-context="vm.syncStartDateChanges"
-         dialog-action="vm.changeSyncStartDate(info, callback)"
-         dialog-title="Update next synchronization date"
-         dialog-action-title="Update"
-         dialog-form="context.mirroringform">
-    <form name="context.mirroringform" class="co-single-field-dialog">
-      Next Synchronization Date:
-      <span id="change_sync_start_date"
-            class="datetime-picker"
-            datetime="vm.syncStartDateChanges.values.sync_start_date"></span>
-    </form>
-  </div>
+          dialog-context="vm.syncStartDateChanges"
+          dialog-action="vm.changeSyncStartDate(info, callback)"
+          dialog-title="Update next synchronization date"
+          dialog-action-title="Update"
+          dialog-form="context.mirroringform">
+      <form name="context.mirroringform" class="co-single-field-dialog">
+        Next Synchronization Date:
+        <span id="change_sync_start_date"
+              class="datetime-picker"
+              datetime="vm.syncStartDateChanges.values.sync_start_date"></span>
```

Comment:
Can you fix all of these binds?

Reply:
This actually doesn't work... Still working on a fix.

Reply:
Ok